### ISSUE TITLE
Make Mekanica have a FE config 

### ISSUE THREAD

**Issue by NailNatail64** [mekanica/mekanica#40]
Question for you. Maybe make a config option to change the green bar to red, change the power cable glow to red instead of green. My point is you can stick in J mode or you can switch to the FE mode. Which is the same but visually different. Since I almost formed mekanism just to change those. Since the green bar is coded in the source it’s not a texture file

**Comment by dizzyd** [comment]
Yes - I like this idea and we were planning to look at switching to FE under the hood anyways. It might be a bit as we've got a lot of other pressing bugs, but we'll keep this on the radar. Thanks!

**Comment by pupnewfster** [comment]
Personally I like the Green, though depending on how it is implemented under the hood it may be interesting (and potentially easy) to make it so that the config has an option for a hex color code, and it uses that in all the places power is rendered.

**Comment by Golrith** [comment]
Also (minor thing), show the power use/generation not as kRF but as actual RF. (yeah, I know it's FE nowadays, but I prefer RF as the terminology :D ) The k and /1000 calculated display just makes the GUI more confusing and messy, and isn't really gaining much screen space.
